openhtf/__init__.py

```diff
@@ -45,6 +46,106 @@ class InvalidTestPhaseError(Exception):
   """Raised when an invalid method is decorated."""
 
 
+def Test(*phases, **metadata):
+  return TestInfo(phases, metadata=metadata)
+
+
+class TestInfo(mutablerecords.Record(
```

Comment:
Fixed it, now it's a dynamic `Test()` with a static `TestInfo()` record inside. I'll clean up the unnecessary properties in another PR since we have enough big PR's in the queue that modifying more files in this PR is just going to make more work for one of us